### ISSUE TITLE
Failed to get convolution algorithm

### ISSUE THREAD

**Issue by lkoll** [serengil/deepface#101]
I am attempting to run some of the example code on some images on my computer. I am operating on Windows 10, Python 3.7 within Spyder and used pip install to install DeepFace. The code I'm attempting to run was taken directly from the main github page, obviously altered paths to account for my image location.

```
from deepface import DeepFace
demography = DeepFace.analyze("Documents/Documents/Masters Degree/Analytics Experience/data/Cleaned Image/01392.png", 
                              actions = ['age', 'gender', 'race', 'emotion'])
Action: age:   0%|          | 0/4 [00:01<?, ?it/s]    
Traceback (most recent call last):

  File "<ipython-input-3-6f17a8bb6a97>", line 2, in <module>
    actions = ['age', 'gender', 'race', 'emotion'])

  File "C:\Users\Luke\anaconda3\lib\site-packages\deepface\DeepFace.py", line 449, in analyze
    age_predictions = age_model.predict(img_224)[0,:]

  File "C:\Users\Luke\anaconda3\lib\site-packages\keras\engine\training.py", line 1169, in predict
    steps=steps)

  File "C:\Users\Luke\anaconda3\lib\site-packages\keras\engine\training_arrays.py", line 294, in predict_loop
    batch_outs = f(ins_batch)

  File "C:\Users\Luke\anaconda3\lib\site-packages\keras\backend\tensorflow_backend.py", line 2715, in __call__
    return self._call(inputs)

  File "C:\Users\Luke\anaconda3\lib\site-packages\keras\backend\tensorflow_backend.py", line 2675, in _call
    fetched = self._callable_fn(*array_vals)

  File "C:\Users\Luke\AppData\Roaming\Python\Python37\site-packages\tensorflow\python\client\session.py", line 1458, in __call__
    run_metadata_ptr)

UnknownError: 2 root error(s) found.
  (0) Unknown: Failed to get convolution algorithm. This is probably because cuDNN failed to initialize, so try looking to see if a warning log message was printed above.
	 [[{{node conv2d_59/convolution}}]]
	 [[activation_8/Softmax/_1379]]
  (1) Unknown: Failed to get convolution algorithm. This is probably because cuDNN failed to initialize, so try looking to see if a warning log message was printed above.
	 [[{{node conv2d_59/convolution}}]]
0 successful operations.
0 derived errors ignored.
```

I have done some searching around online but haven't found any fixes, or anything related to DeepFace specifically with this error. I am using tensorflow without issue in other programs.

**Comment by serengil** [comment]
Could you set cuda visible devices as null before importint deepface first.

```
import os
os.environ["CUDA_VISIBLE_DEVICES"]=""
```

Besides, what version of tensorflow and keras you are using. I tested it in tf 1.9.0 and keras 2.2.0.

**Comment by lkoll** [comment]
Setting cuda visible devices to null with your code had the same result.

I was using tf 2.2 and keras 2.2.4

When you say tf 1.9.0, where do you get that from? When I pip uninstall my current tf and try to install that I get the below

```
C:\Windows\system32>pip install tensorflow==1.9.0
ERROR: Could not find a version that satisfies the requirement tensorflow==1.9.0 (from versions: 1.13.0rc1, 1.13.0rc2, 1.13.1, 1.13.2, 1.14.0rc0, 1.14.0rc1, 1.14.0, 1.15.0rc0, 1.15.0rc1, 1.15.0rc2, 1.15.0rc3, 1.15.0, 1.15.2, 1.15.3, 1.15.4, 2.0.0a0, 2.0.0b0, 2.0.0b1, 2.0.0rc0, 2.0.0rc1, 2.0.0rc2, 2.0.0, 2.0.1, 2.0.2, 2.0.3, 2.1.0rc0, 2.1.0rc1, 2.1.0rc2, 2.1.0, 2.1.1, 2.1.2, 2.2.0rc0, 2.2.0rc1, 2.2.0rc2, 2.2.0rc3, 2.2.0rc4, 2.2.0, 2.2.1, 2.3.0rc0, 2.3.0rc1, 2.3.0rc2, 2.3.0, 2.3.1)
ERROR: No matching distribution found for tensorflow==1.9.0
```

**Comment by serengil** [comment]
tensorflow 1.9.0 is available for python 3.6

**Comment by lkoll** [comment]
So you just can't use pip to install it or what?
